app/scripts/controllers/paymentModal.js

```diff
@@ -326,6 +326,7 @@ angular.module('confRegistrationWebApp')
     /*
     Returns false is payment if payment and refund channel are credit card and current time is less than 24 hours from
     payment time.  If a refund is issued before the payment settles, authorize.net will do a full refund regardless of amount.
+    ***Authorize.net no longer used, but payment is still used so keeping code below for now.***
     */
     $scope.isPartialRefundAvailable = function(payment, refundChannel) {
```

Comment:
@hlbraddock Do you know if the idea of partial refunds still exists for TSYS? This seems to have some specific rules preventing partial refunds for 24 hours for Authorize.net.